web-console/knikubevirt/services-in-overview/services-in-overview.md

```diff
@@ -0,0 +1,11 @@
+# Services in overview
+
+How do display services on a resource detail view
```

Comment:
How about rephrasing this to explain it from a user's perspective. Why would a user want to see these services?

Reply:
You'll probably catch this in the when re-writing but "How do" should be "How to" :)